streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -328,7 +328,7 @@ public void shouldCommitAfterTheCommitInterval() {
 
     @SuppressWarnings({"ThrowableNotThrown", "unchecked"})
     private TaskManager mockTaskManagerCommit(final Consumer<byte[], byte[]> consumer, final int numberOfCommits, final int commits) {
-        final TaskManager taskManager = EasyMock.createMock(TaskManager.class);
+        final TaskManager taskManager = EasyMock.createNiceMock(TaskManager.class);
```

Comment:
Why?

Reply:
It has one more functions that is not focus of the test now.